code/lib/preview-web/src/render/StoryRender.ts

```diff
@@ -24,6 +25,8 @@ import { Render, RenderType } from './Render';
 
 const { AbortController } = global;
 
+export type MaybePromise<T> = Promise<T> | T;
```

Comment:
Why did you move this in here I wonder?

Reply:
Yeah it got migrated here when StoryRender was pulled out of PreviewWeb I think. Could just be in Preview, so I moved it there now.